### DISCUSSION TITLE
在哪里定义环境 projects

### DISCUSSION THREAD

**Discussion by herozen97** [sjtug/SJTUThesis#1044] [提问] [answered]
模板中最后缺少projects 这个单元用以表达 “参与的项目”。
所以参考其他元素，类似的 \input{contents/projects}
然后文件 projects.tex 中 begin{projects}. 
报错不能识别 projects 环境。
但是我在文件目录中没有找到 abstract, achievements 这些环境的定义。
所以请问我该如何操作呢。
谢谢！

**Comment by herozen97** [comment, marked as answer]
终于找到 sjtuthesis.cls了
参考achievements 写了project的环境

**Comment by LogCreative** [comment]
可以直接 
```latex
\begin{achievements}[参与的项目]

\end{achievements}
```
